Azure/azure-key-vault/ingest/parser.yml

```diff
@@ -0,0 +1,33 @@
+name: azure-key-vault
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: "event"
+          event.category: "database"
+          event.dataset: "keyvault"
+          event.outcome: "{{json_event.message.resultSignature}}"
```

Comment:
The value of `event.outcome` is restricted: https://www.elastic.co/guide/en/ecs/8.6/ecs-allowed-values-event-outcome.html
I suggest setting this field according to the resultSignature: (`failure` for `Forbidden`, `success` for `OK`).